core/src/main/java/com/amplifyframework/util/FieldFinder.java

```diff
@@ -45,21 +46,40 @@ private FieldFinder() {
      * @return set of fields
      */
     @NonNull
-    public static List<Field> findFieldsIn(@NonNull Class<?> clazz) {
+    public static List<Field> findModelFieldsIn(@NonNull Class<?> clazz) {
```

Comment:
@saltonmassally, @richardmcclellan, @jamesonwilliams I know this wasn't flagged on the original PR but just wanted to double check we're good changing this public method name since it would be a breaking change if anyone was currently using it?

Reply:
No one should be using our `FieldFinder`; it's an internal utility in the context of the project.

Reply:
@jamesonwilliams makes sense.  I just removed my commit entirely for now.